data/json/items/migration.json

```diff
@@ -20,6 +20,12 @@
     "replace": "battery_car"
   },
   {
+    "id": "gold_small",
+    "type": "MIGRATION",
+    "replace": "gold",
+    "charges": 1
```

Comment:
I don't know for certain but its a reasonable assumption and has the significant advantage that we don't need any new migration code.